Sources/SwiftParser/Parser.swift

```diff
@@ -29,11 +29,12 @@
 /// from a bad parse should be left `nonmutating` to indicate that they do not
 /// consume tokens.
 ///
-/// Token consumption is generally conditional via ``TokenConsumer/consume(if:)``
-/// or unconditional via `consumeAnyToken()`. During parsing, it is
-/// also useful to assert that the current token matches some expected structure
-/// via ``TokenConsumer/eat(_:)``, which acts like ``TokenConsumer/consume(if:)``,
-/// but asserts if the parsed token did not match the expected kind.
+/// Token consumption is generally either unconditional via ``TokenConsumer/consumeAnyToken()``
```

Comment:
Good idea. I've added it to the end of this section. The two methods are used with roughly equal frequency in the codebase, and I think it's better to point folks at at-eat than consume(if:) since they'll often exchange the latter for the former when doing recovery or parsing more complex elements.